pyproject.toml

```
@@ -45,7 +45,7 @@ lightkube = "^0.13.0"
 pytest = "^7.3.1"
 pytest-mock = "^3.11.1"
 pytest-operator = "^0.27.0"
-juju = "2.9.44.0 || 3.2.0.1"
+juju = "3.2.0.1"
```

maybe, can be separate PR
```suggestion
juju = "3.2.2"
```

---

Will commit in this one  if you don't mind